lib/src/lib/table/components/table-filter-menu/table-filter-menu.component.ts

```diff
@@ -49,32 +48,23 @@ import { MatButtonModule } from '@angular/material/button';
   ],
 })
 export class Lab900TableFilterMenuComponent {
-  public readonly tableCells$: Observable<TableCell[]>;
-  public readonly visibleCells$: Observable<TableCell[]>;
-  public readonly hiddenCells$: Observable<TableCell[]>;
+  private readonly tableService = inject(Lab900TableService);
+  public readonly tableCells = computed(() =>
+    this.tableService
+      .columns()
+      .filter((cell: TableCell) => !cell.alwaysVisible),
```

Comment:
maybe i'm missing something, but isn't this check wrong?
if a cell is `alwaysVisible`, it will be filtered out

Reply:
If they are always visible they should not be part of the filter menu

Reply:
Will rename for future reference